client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -40,6 +42,9 @@ function ReaderSubscriptionListItem( {
 	showFollowedOnDate,
 	isFollowing,
 	railcar,
+	isLoggedIn,
+	/* eslint-disable no-shadow */
+	registerLastActionRequiresLogin,
```

Comment:
Alternatively we could get away from shadowing in the destructure by renaming inline here like `registerLastActionRequiresLogin: newName,` where newName could be something like registerRequiringLoging or something

Reply:
Added registerLastActionRequiresLoginProp as the newName here and it works ok